static/app/views/replays/replays.tsx

```diff
@@ -34,12 +34,19 @@ type Props = AsyncView['props'] &
 class Replays extends React.Component<Props> {
   getEventView() {
     const {location, selection} = this.props;
+    const {query} = location;
+
+    let sort = '-timestamp';
+    if (query.sort) {
+      sort = query.asc ? query.sort : `-${query.sort}`;
```

Comment:
Instead of 2 query params (`asc` and `sort`), can we condense this down to just `sort`? This mimics the current behavior in Discover.